docs/scratch/modifications.md

```diff
@@ -0,0 +1,92 @@
+# Scratch Modifications
+
+As part of this project, we modified scratch.
+This file lists and describes the changes.
+
+## Edit and Solve mode
+
+We extended the scratch GUI component to additionally accept a `canEditTask` property which, if disabled, deactivates the following built-in sratch features:
+- the button to load extensions is not shown
+- the buttons to add new sprites is not shown
+- the buttons to load stage sprites is not shown
+- the buttons to delete, duplicate or export sprites is not shown
+- the rendered stage is made non-interactive using `pointer-events: none`
+
+In the following we will use "Task editing mode" (or "Edit mode") to denote `canEditTask=true` and "Task solve mode" (or "Solve mode") to denote `canEditTask=false`.
```

Comment:
If you define it as `Edit mode` (capitalized), be careful not to switch to "edit mode" later, as it could lead to misinterpretation (edit as verb, mode as object)